src/lsst/rsp/utils.py

```diff
@@ -41,7 +41,7 @@ def format_bytes(n: int) -> str:
         return "%0.2f MB" % (n / 1e6)
     if n > 1e3:
         return "%0.2f kB" % (n / 1000)
-    return "%d B" % n
```

Comment:
I wonder if we ought to put the whole thing in new-style format strings.  Having just one of them seems weird.

Reply:
Done.